File: python/srednia.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-  

def pobierzOceny():
    oceny = []
    ocena = int(input('Podaj ocenę: '))
    while ocena > 0:
        if ocena < 7:
            oceny.append(ocena)
        else:
            print('Błędna ocena!')
        ocena = int(input('Podaj ocenę: '))
    return oceny
    
def liczSrednia(oceny):
    print(sum(oceny) / len (oceny))

def main(args):
    oceny = pobierzOceny()
    liczSrednia(oceny)
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: CPP/srednia.cpp
/*
 * srednia.cpp
 */


#include <iostream>
using namespace std;


int main(int argc, char **argv)
{
    int oceny = 10;
    int ocena;
    cout << "Podaj ocenę: " << endl;
    cin >> ocena;
    while (ocena > 0){
        if (ocena < 7){
            
        } else {
            printf("Błędna liczba!");
            cin >> ocena;
            return oceny;
        }
    }
    cout << "Średnia wynosi: " << ocena / oceny << endl;
    
	return 0;
}

